ohpc.yaml

```diff
@@ -7,3 +7,4 @@
     - { name: 'compute_build_vnfs', tags: 'compute_build_vnfs' }
     - { name: 'compute_build_nodes', tags: 'compute_build_nodes' }
     - { name: 'nodes_vivify', tags: 'nodes_vivify' }
+    - { name: 'ohpc_matlab', tags: 'ohpc_matlab' }
```

Comment:
Is there a way to make such components optional?  Either by setting a flag in groups_var/all or an explicit run of this role when matlab support is desired?  What would be the recommended approaches to accomplish this?